hydra-tui/src/Hydra/TUI.hs

```diff
@@ -134,6 +116,39 @@ makeLensesFor
   ]
   ''HeadState
 
+--
+
+-- * User feedback handling
+
+--
+
+severityToAttr :: Severity -> AttrName
+severityToAttr = \case
+  Success -> positive
+  Info -> infoA
+  Error -> negative
+
+infoA :: AttrName
+infoA = "info"
+
+positive :: AttrName
+positive = "positive"
+
+negative :: AttrName
+negative = "negative"
+
+own :: AttrName
+own = "own"
```

Comment:
Just a note on ordering (which maybe got derailed over time and why you saw the need to move things)

The definitions above here are only about rendering, whereas the three definitions below are about handling events & updating state. I used to have all the rendering stuff in one location in the file, probably should have moved it into individual files to make that organization clear.

No need to change now. Probably can use some general overhaul another time.